packages/apps/explorer/src/constants/network.ts

```diff
@@ -1,15 +1,17 @@
 export const networkConstants = {
-  mainnet01: {
+  mainnet: {
     networkId: 'mainnet01',
     label: 'Mainnet',
+    slug: 'mainnet',
     chainwebUrl: 'api.chainweb.com',
     graphUrl: 'https://graph.kadena.network/graphql',
     wsGraphUrl: 'https://graph.kadena.network/graphql',
     explorerUrl: 'https://explorer.kadena.io/',
   },
-  testnet04: {
+  testnet: {
     networkId: 'testnet04',
     label: 'Testnet',
+    slug: 'testnet',
     chainwebUrl: 'api.testnet.chainweb.com',
     graphUrl: 'https://graph.testnet.kadena.network/graphql',
     wsGraphUrl: 'https://graph.testnet.kadena.network/graphql',
```

Comment:
We don't do anything specific that we need to use a Record<string,string> so you can also convert it to an array with `type Array<INetwork>`